src/services/calendar/hooks/index.js

```diff
@@ -9,32 +9,33 @@ const { courseModel } = require('../../user-group/model');
  * */
 const persistCourseTimesEvent = (hook) => {
 	const courseService = hook.app.service('courses');
-	return Promise.all(hook.result.map((event) => {
-		if (event['x-sc-courseId']) {
-			const courseId = event['x-sc-courseId'];
-			const courseTimeId = event['x-sc-courseTimeId'];
+	return Promise.all(
+		hook.result.map((event) => {
```

Comment:
Codacy found an issue: [Expected to return a value at the end of arrow function.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)